ProjectLighthouse.Servers.Website/Pages/Partials/SlotCardPartial.cshtml

```diff
@@ -110,6 +110,14 @@
                     <i class="cog icon" style="margin: 0"></i>
                 </a>
             }
+            
+            @if (user != null && !mini && (user.UserId != Model.CreatorId))
```

Comment:
Are moderators/admins supposed to be able to flag levels as duplicates or just the level creator?

Reply:
Anybody who is not the level creator should be able to flag levels as duplicate.